BlueprintUICommonControls/Sources/ScrollView.swift

```diff
@@ -17,6 +17,7 @@ public struct ScrollView: Element {
     public var showsHorizontalScrollIndicator: Bool = true
     public var showsVerticalScrollIndicator: Bool = true
     public var pullToRefreshBehavior: PullToRefreshBehavior = .disabled
+    public var shouldEndEditingOnTap = false
```

Comment:
Would it make more sense to expose the keyboardDismissMode here? That would also allow callers to utilize `interactive` if they wish.

Reply:
I originally was going to do that but I figured Blueprint is trying to abstract away UIKit as much as possible. But then again that is just my assumption.

I tried out `.interactive` and it doesn't seem very useful.

But I'm open to either way.

Reply:
I think in this case, it makes sense to expose the entire enum: If someone needs or wants interactive dismissal (which makes sense – eg to mirror a UI similar to messages), we'd need to introduce a second property which overlaps with this one and change how `shouldEndEditingOnTap` is applied to check if interactive dismissal is enabled – which isnt as clean.

In terms of abstraction: Blueprint does generally abstract away the complexities of UIKit when it makes sense (eg, see `PullToRefreshBehavior` as a good example – reducing the complexities of defining PTR down to one enum, versus dealing with views and target/action callbacks). In this case, because the enum is a 1:1 passthrough, exposing it is fine.

Reply:
Changed property type to UIScrollView.KeyboardDismissMode. It is now passing through to the underlying UIScrollView.